### ISSUE TITLE
Fatal error: Uncaught PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'ip' in 'where clause'

### ISSUE THREAD

**Issue by ltdev22** [cartalyst/sentinel#541]
### Your Environment

- OS and Version: Docker
- PHP Version: 7.3
- Sentinel Version: ^5.0

### Expected behaviour

Trying to authenticate with my credentials on a php app

```

use Cartalyst\Sentinel\Native\Facades\Sentinel;

$data = $request->getParsedBody(); // array contains only email and password

if (!$user = Sentinel::authenticate($data)) {
    dd('Incorrect credentials');
}
dd($user);
```

### Actual behaviour

Getting the following error on screen when I make the request

```
Fatal error: Uncaught PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'ip' in 'where clause' in /var/www/html/vendor/illuminate/database/Connection.php:331 Stack trace: #0 /var/www/html/vendor/illuminate/database/Connection.php(331): PDO->prepare('select * from `...') #1 /var/www/html/vendor/illuminate/database/Connection.php(664): Illuminate\Database\Connection->Illuminate\Database\{closure}('select * from `...', Array) #2 /var/www/html/vendor/illuminate/database/Connection.php(631): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure)) #3 /var/www/html/vendor/illuminate/database/Connection.php(339): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure)) #4 /var/www/html/vendor/illuminate/database/Query/Builder.php(2302): Illuminate\Database\Connection->select('select * from `...', Array, true) #5 /var/www/html/vendor/illuminate/database/Query/Builder.php(2290): Illuminate\Database\Query\Builder->runSelect() #6 /var/www/html/vendor/i in /var/www/html/vendor/illuminate/database/Connection.php on line 671
```

I'm not sure if this is related to docker, but this is the docker-compose file
```
mysql:
        image: mysql:5.7
        volumes:
            - dbdata:/var/lib/mysql
        ports:
            - 4306:3306
        networks:
            - appnet
        env_file:
            - ./.env

// .env file

DB_HOST=127.18.0.2
DB_DATABASE=my_db
DB_USERNAME=my_username
DB_PASSWORD=my_password

MYSQL_ROOT_PASSWORD=root
MYSQL_DATABASE="${DB_DATABASE}"
MYSQL_USER="${DB_USERNAME}"
MYSQL_PASSWORD="${DB_PASSWORD}"

```

**Comment by suwardany** [comment]
What's your database schema?

**Comment by ltdev22** [comment]
id int(11) pk autoincrement
first_name varchar(255)
last_name varchar(255)
email varchar(255)
password (255)
ip varchar(255)
last_login_at datetime
created_at datetime
updated_at datetime

**Comment by suwardany** [comment]
`ip` is a column on the throttle table, i would suggest you checkout `.sql` files inside the `schema` directory of this repository.

**Comment by ltdev22** [comment]
Yes I have created the tables based on schema/mysql.sql, but I was getting this error even before. So I tried to add an extra ip column in `users`  table to test if it would fix the issue

**Comment by ltdev22** [comment]
Ok, so I've re-run the schema/mysql.sql this time by copying all queries from this file (last time I created the tables manually on my gui tool). I had to change the default values for created_at and updated_at from 
`created_at` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
`updated_at` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
to 
`created_at` timestamp NOT NULL DEFAULT NOW(),
 `updated_at` timestamp NOT NULL DEFAULT NOW(),
as it was complaining for invalid default value and then when I run the queries worked. Also when I tried the authenticate script I've put on my topic it worked. Not sure if this was the issue, but now seems its working fine.
